src/components/CompareResults/RevisionRow.tsx

```diff
@@ -181,7 +181,7 @@ function RevisionRow(props: RevisionRowProps) {
           <div className='graph-link-button-container'>
             <IconButton aria-label='graph link' size='small'>
               <Link href={graphLink} target='_blank'>
-                <TimelineIcon />
+                <TimelineIcon titleAccess='link to treeherder graph' />
```

Comment:
The title should contain information about the image itself, not about the link. I'd be interested to know what the axe error was in this case, can you please explicit? Then I may be able to suggest something different.

Reply:
I'd suggest to use `aria-label` on the Link instead, as suggested in https://mui.com/material-ui/icons/#semantic-icons
On the Link we can provide a more contextual information, in that case `Go to Treeherder graph` maybe. We can probably remove the aria-label on the IconButton above, maybe even add `role="presentation"` so that it doesn't have any meaning to screen reader users.